lib/cocoapods/command/repo/lint.rb

```diff
@@ -35,16 +35,20 @@ def initialize(argv)
         #
         def run
           if @name
```

Comment:
this can only assign once,  by assigning the result of the conditional to `sources`